openslides/users/views.py

```diff
@@ -169,6 +170,20 @@ def destroy(self, request, *args, **kwargs):
         self.perform_destroy(instance)
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    def update(self, request, *args, **kwargs):
+        # Collect old and new permissions to get the difference.
+        permission_names = request.data['permissions']
+        if isinstance(permission_names, str):
+            permission_names = [permission_names]
+
+        permissionSerializer = PermissionRelatedField(read_only=True)
+        given_permissions = [
+            permissionSerializer.to_internal_value(data=perm) for perm in permission_names]
+        group = Group.objects.get(pk=int(kwargs['pk']))
+        inform_changed_permissions(group, given_permissions)
```

Comment:
I think the information should be send after the update was successful. So first run super(). Then run the code to check for objects to send.